app/src/main/java/zapsolutions/zap/connection/LndConnection.java

```diff
@@ -73,7 +73,11 @@ private void readSavedConnectionInfo() {
         String certificateBase64UrlString = mConnectionInfo[2];
         byte[] certificateBytes = BaseEncoding.base64Url().decode(certificateBase64UrlString);
 
-        mSSLFactory = CustomSSLSocketFactory.create(certificateBytes);
+        try {
```

Comment:
Yes, it crashed if it was unable to get a vailid certificate from the supplied bytes.
Actually I now ceck this already on the LndConnect String parser so it might not be needed again here.
The reason why I still added it here is because I don't know if it also crashes if the certificate gets invalid when it expires.